jekyll/_cci2/rerun-failed-tests.adoc

```diff
@@ -230,6 +230,39 @@ You can also add it to your `jest.config.js` file by following these link:https:
 +
 NOTE: Ensure that you are using version 1.34.2 or later of Playwright. Earlier versions of Playwright may not output JUnit XML in a format that is compatible with this feature.
 
+[#configure-a-job-running-gradle-tests]
+=== Configure a job running Gradle tests
+
+. Modify your test command to use `circleci tests run`:
+
+# Get list of classnames of tests that should run on this node
```

Comment:
Is this intended to be a comment in the code block @sebastian-lerner ?

Reply:
sorry, this is intended to be in the code block: `# Get list of classnames of tests that should run on this node"`